docs/ko/docs/advanced/response-cookies.md

```diff
@@ -0,0 +1,52 @@
+# 응답 쿠키
+
+## `Response` 매개변수 사용하기
+
+*경로 작업 함수*에서 `Response` 타입의 매개변수를 선언할 수 있습니다.
+
+그런 다음 해당 *임시* 응답 객체에서 쿠키를 설정할 수 있습니다.
+
+```Python hl_lines="1  8-9"
+{!../../docs_src/response_cookies/tutorial002.py!}
+```
+
+그런 다음 필요한 객체(예: `dict`, 데이터베이스 모델 등)를 반환할 수 있습니다.
+
+그리고 `response_model`을 선언했다면 반환한 객체를 필터링하고 변환하는 데 여전히 사용됩니다.
+
+**FastAPI**는 그 *임시* 응답에서 쿠키(또한 헤더 및 상태 코드)를 추출하고, 반환된 값이 포함된 최종 응답에 이를 넣습니다. 이 값은 `response_model`로 필터링됩니다.
+
+또한 종속성에서 `Response` 매개변수를 선언하고, 해당 종속성에서 쿠키(및 헤더)를 설정할 수도 있습니다.
+
+## `Response`를 직접 반환하기
+
+코드에서 `Response`를 직접 반환할 때도 쿠키를 생성할 수 있습니다.
+
+이를 위해 [Response를 직접 반환하기](response-directly.md){.internal-link target=_blank}에서 설명한 대로 응답을 생성할 수 있습니다.
+
+그런 다음 쿠키를 설정하고 반환하면 됩니다:
+Python hl_lines="10-12"
```

Comment:
could you check the new created documentation? 
it is broken from here.

Reply:
``` 표시 추가해서 변경했습니다!